crypto/x509/x_pubkey.c

```diff
@@ -219,8 +219,10 @@ EVP_PKEY *X509_PUBKEY_get0(X509_PUBKEY *key)
 EVP_PKEY *X509_PUBKEY_get(X509_PUBKEY *key)
 {
     EVP_PKEY *ret = X509_PUBKEY_get0(key);
-    if (ret != NULL)
-        EVP_PKEY_up_ref(ret);
+    if (ret != NULL && !EVP_PKEY_up_ref(ret)) {
+        X509err(X509_F_X509_PUBKEY_GET0, ERR_R_INTERNAL_ERROR);
```

Comment:
Wrong F code here. You can just make it 0, since the F codes are no longer used.

Reply:
Incredibly Travis found this also.  So I will fix this, but it means
this will not cherry-pick cleanly to 1.1.1, sigh...